common-libs/internal-api-clients/src/index.ts

```diff
@@ -19,3 +19,5 @@ export {
   CloudWalletApiService,
   VaultMigrationApiService,
 }
+
+export type { DidAuthAdapterType } from '@affinidi/tools-openapi'
```

Comment:
Is this just for convenience?

Reply:
To avoid adding unneeded dependency on `tools-openapi` to `wallet-core-sdk` which implements `DidAuthAdapterType`